core/app/models/spree/option_type.rb

```diff
@@ -13,7 +13,7 @@ class OptionType < Spree::Base
     has_many :prototypes, through: :option_type_prototypes, class_name: 'Spree::Prototype'
 
     with_options presence: true do
-      validates :name, uniqueness: { allow_blank: true }
+      validates :name, uniqueness: { case_sensitive: false, allow_blank: true }
```

Comment:
This might be not enough. Probably need to do the following also.

```ruby
execute <<-SQL
  CREATE UNIQUE INDEX option_type_name_idx ON option_types (LOWER(name));
SQL
```

And all other places also!